test/extension/config/tilt/extensionconfig.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: runtime.cluster.x-k8s.io/v1alpha1
+kind: ExtensionConfig
+metadata:
+  annotations:
+    runtime.cluster.x-k8s.io/inject-ca-from-secret: capv-test-extension/capv-test-extension-webhook-service-cert
+  name: capv-test-extension
+spec:
+  clientConfig:
+    service:
+      name: capv-test-extension-webhook-service
+      namespace: capv-test-extension # Note: this assumes the test extension get deployed in the default namespace defined in its own runtime-extensions-components.yaml
+      port: 443
+  namespaceSelector:
+    matchExpressions:
+      - key: kubernetes.io/metadata.name
+        operator: In
+        values:
+          - default # Note: this assumes the test extension is used by Cluster in the default namespace only
```

Comment:
q: is this true for our tests?

Reply:
this is not true for our tests, this is only for usage with tilt.